modules/api/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "serverless_lambda_api_invoke_arn" {}
+variable "serverless_lambda_api_name" {}
+variable "cloudwatch_serverless_access_arn" {}
+variable "cloudwatch_serverless_logs_role_arn" {}
```

Comment:
![image](https://github.com/mohamed97-ba/agyla_labs_infra/assets/47115488/86eee241-f450-4b48-963a-b735814a9b7d)
this means that the file needs an end of line